code/modules/heavy_vehicle/mecha.dm

```diff
@@ -250,6 +250,8 @@
 /mob/living/heavy_vehicle/GetIdCard()
 	return access_card
 
+/// Checks if mech can be powered on/off, sends message to pilot if failed
+/// If remotely called, messages and other hud elements are called on the exosuit itself to prevent wierdness, and errors are handled in `handle_hear_say()`
```

Comment:
This isn't telling me what `var/remote` means, and what it expects to get, is it a boolean? Does it use defines? Etc.